public/js/surveyBuilderForm.js

```diff
@@ -0,0 +1,190 @@
+$(function() {
+    var addQuestionButton = $('#add-question')
+
+    addQuestionButton.prop("disabled", true)
+
+    var $type = $('#input-selector')
+
+    $type.change(function () { // attaching a listener for the question input type being changed
+
+        //checking that it's being changed from the text input type to a non-text input type
+        if ($('#question-options').length < 1 && $type.val() !== 'text-input') {
+            addOptionsCreator($('#input-container')) // brings up the multiple choice options controls
```

Comment:
should you comment a function call? surely the comment on the function should be enough?